app/main/views.py

```diff
@@ -163,7 +163,9 @@ def index():
 @main.route('/standings/')
 def standings():
     years = sorted(set(
-        int(date.year) for (date,) in Race.query.with_entities(Race.date).all()),
+        int(date.year) for (date,) in Race.query.with_entities(Race.date)\
+                                                .filter_by(points_race=True)\
+                                                .all()),
```

Comment:
Unrelated to this change, but the all() isn't necessary either -- creating the set takes care of evaluating the query.